loops/loop30.go

```diff
@@ -0,0 +1,24 @@
+// divisibility by 2 and 6
+
+package main
+
+import "fmt"
+
+func main() {
+	numbers := []int{2,3,5,9,18,20,25,27,29,35,46,}
+	for i := range numbers {
+		if numbers[i] % 2 == 0 {
+			if numbers[i] % 3 == 0 {
+				fmt.Printf("%d is divisible by 2 and 6\n", numbers[i])
```

Comment:
You mean '2 and 3' here I guess.

Reply:
No it's 2 and 6 (well 3 also. but in this if a number is divisible by 2 and 3, it's divisible by 6.)

Reply:
Ok, so that would be 2, 3 and 6 then. :)